checkstyle/checkstyle.xml

```diff
@@ -9,7 +9,7 @@
 
   <!-- header -->
   <module name="RegexpHeader">
-    <property name="header" value="/\*\*\n * Copyright \(c\)"/>
+    <property name="header" value="/\*\*?\n * Copyright \(c\)"/>
```

Comment:
The PR description no longer reflects this change. Could you please update, @TisonKun ?

Reply:
Updated